### ISSUE TITLE
[ERROR- Couldn't parse file.] when openeing EP1 Challenge mode quests

### ISSUE THREAD

**Issue by zharima** [DaanVandenBosch/phantasmal-world#16]
Hi, thank you for your work here, it has been instrumental in helping me parse the quest files way easier than using qedit.
However it seems all the ep1 cmode quests return the error  "Couldn't parse file. " when trying to open these files.

All the other quests are fine, also ep2 cmode opens fine, just something with ep1 cmode files, they do open in qedit correctly so I know the files are not corrupt.

No rush to fix, just thought I should report this error, thanks!

![Screenshot from 2023-06-11 21-07-51](https://github.com/DaanVandenBosch/phantasmal-world/assets/122275609/baedbe07-c70a-4e03-85b1-f322aa772231)

**Comment by DaanVandenBosch** [comment]
Yeah, that's a known problem. But thanks for reporting it so people can see the status here. I should fix it at some point, but haven't even figured out what the issue is. I'm also not working on phantasmal.world much at the moment, so it might take a while before it's fixed, sorry.

**Comment by warmonipa** [comment]
#38 fixed
